examples/sync/README.md

```diff
@@ -107,11 +115,16 @@ curl http://localhost:9090/metrics
 
 1. **Server Setup**: Server starts, populates database with initial operations, and listens for connections
 2. **Continuous Operation Generation**: Server continuously adds new operations at the specified interval
-3. **Client Connection**: Client establishes connection to server
-4. **Initial Sync Target**: Client requests server metadata to determine initial sync target (inactivity floor, size, and digest of the server's database)
-5. **Dynamic Target Updates**: Client periodically requests target updates during sync to handle new operations added by the server
-6. **Completion**: Client continues until all operations are applied and state matches server's target
-7. **Cleanup**: Client disconnects and stops; Server keeps running and adding operations
+3. **Client Startup**: Client starts continuous sync process
+4. **Sync Iteration Loop**: For each sync iteration:
+   - **Database Initialization**: Client opens a new database (or reopens existing one)
+   - **Connection**: Client establishes connection to server
+   - **Initial Sync Target**: Client requests server metadata to determine sync target (inactivity floor, size, and root digest)
+   - **Dynamic Target Updates**: Client periodically requests target updates during sync to handle new operations added by the server
+   - **Sync Completion**: Client continues until all operations are applied and state matches server's target
+   - **Database Closure**: Client closes the database to prepare for next iteration
+   - **Wait Period**: Client waits for the configured sync interval before starting next iteration
+5. **Continuous Operation**: This process continues indefinitely, allowing the client to stay synchronized with the ever-changing server state
```

Comment:
I had imagined, originally, that we'd "never end the sync" procedure (while still allowing db access) rather than continuously restarting the entire thing but I think that's a significant amount of extra work.

This was a pattern I thought could be useful for indexing-focused nodes that don't want to (or can't) run consensus (and just want to sync state proofs).

Reply:
I think what you have is definitely worth keeping as part of the example in any case 👍